igvm/settings.py

```diff
@@ -77,7 +77,8 @@
 MAC_ADDRESS_PREFIX = (0xCA, 0xFE, 0x01)
 
 try:
-    IGVM_IMAGE_URL = environ['IGVM_IMAGE_URL']
+    # IGVM_IMAGE_URL = environ['IGVM_IMAGE_URL']
+    IGVM_IMAGE_URL = 'http://foreman-master.ndco.ig.local:8080/new_images/{image}'
```

Comment:
🤔

Reply:
Just for now get get tests twice as fast. Will be gone, see that it is a separate commit.